sql/core/src/test/resources/sql-tests/inputs/show-tables.sql

```diff
@@ -17,6 +17,12 @@ SHOW TABLES LIKE 'show_t1*|show_t2*';
 SHOW TABLES IN showdb 'show_t*';
 SHOW TABLES IN showdb LIKE 'show_t*';
 
+-- SHOW CACHED TABLES
+CACHE TABLE show_t1;
```

Comment:
Could you add test cases for `AS query`?

Reply:
added.